lib/gnarails/cli/application.rb

```diff
@@ -165,26 +26,13 @@ class Application < Thor
         You should also be able to pass any other arguments you would expect to
         be able to when generating a new rails app.
       LONGDESC
-      def new(name)
-        Kernel.system "rails new #{name} #{cli_options(options)}"
+      def new
+        Kernel.system "rails new #{cli_options}"
```

Comment:
I think we still need the name to be able to set up the rails app with the right name for the directory and the default app name.

Reply:
I thought that was something rails did by default? Maybe I'm misunderstanding but I imagine that users will, instead of doing:
```
rails new --options my_new_app
```
they're going to do
```
gnarails --options my_new_app
```
and so we should hopefully pass `my_new_app` through to `rails new` and it'll make the new directory and set the default app name and stuff. I haven't been able to test this since I'm not sure how to compile the gem locally but I could definitely test it!

Reply:
> I haven't been able to test this since I'm not sure how to compile the gem locally

To run your local changes, you should be able to use the [exe](https://github.com/TheGnarCo/gnarails/blob/d2209ae8dd963ee539f497dbd32798bad5d92154/exe/gnarails) included in the repo. You can reference [this script](https://github.com/TheGnarCo/gnarails/blob/d2209ae8dd963ee539f497dbd32798bad5d92154/bin/generate-test-app.sh#L2) which is used by the [test setup script](https://github.com/TheGnarCo/gnarails/blob/d2209ae8dd963ee539f497dbd32798bad5d92154/bin/test-setup.sh#L1), which is used by the [integration test](https://github.com/TheGnarCo/gnarails/blob/d2209ae8dd963ee539f497dbd32798bad5d92154/spec/integration_spec.rb#L9).